native/src/components/__tests__/MalteHelpFormOffer.spec.tsx

```diff
@@ -0,0 +1,411 @@
+import { userEvent, waitFor } from '@testing-library/react-native'
+import React from 'react'
+import mockSafeAreaContext from 'react-native-safe-area-context/jest/mock'
+
+import { InvalidEmailError, OfferModel, submitMalteHelpForm } from 'shared/api'
+
+import renderWithTheme from '../../testing/render'
+import MalteHelpFormOffer from '../MalteHelpFormOffer'
+
+jest.mock('react-i18next')
+jest.mock('shared/api', () => ({
+  ...jest.requireActual('shared/api'),
+  submitMalteHelpForm: jest.fn(),
+}))
+jest.mock('react-native-safe-area-context', () => mockSafeAreaContext)
+const mockShowSnackbar = jest.fn()
+jest.mock('../../hooks/useSnackbar.ts', () => ({
+  __esModule: true,
+  default: () => mockShowSnackbar,
+}))
+
+// Typescript shenanigans to be able to use mockImplementation in the tests
+const mockedSubmitMalteHelpForm = submitMalteHelpForm as jest.MockedFunction<typeof submitMalteHelpForm>
```

Comment:
:wrench: This should not be needed. Instead, you can just do `mocked(submitMalteHelForm).mockImplementation(...)`.

Reply:
Amazing, thank you! That is so much better!